packages/components/modules/messages/graphql/mutations/CreateChatRoom.ts

```
@@ -0,0 +1,64 @@
+
+  const commit = (config: UseMutationConfig<CreateChatRoomMutation>) =>
+    commitMutation({
+      ...config,
+      onCompleted: (response, errors) => {
+        errors?.forEach((error) => {
+          sendToast(error.message, { type: 'error' })
+        })
+        setChatRoom({
+          id: response?.chatRoomCreate?.room?.node?.id,
+          participants: response?.chatRoomCreate?.room?.node?.participants?.edges?.map(
+            (edge) => edge?.node && edge.node.id,
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)